js/application.js

```diff
@@ -0,0 +1,81 @@
+
+
+$(document).ready(function(){
+
+
+   function createitem(){
+
+
+      var product_name = $(".product_name").val();
+      var product_price = $(".product_price").val();
+
+      console.log(product_name);
+      console.log(product_price);
+
+      if ($.isNumeric(product_price)===false){
+         alert("Please enter a number");
+      }else if (product_name===" "){
+        alert('Please enter the product name');
+      } else{
+        var item=''+ '<div class="item row" >'+ '<div class="productname col-xs-3" >'+product_name+'</div>'+ '<div class="unitpriced col-xs-3" >'+'$'+product_price+'</div>' + '<div class="quantity col-xs-4" >' + '<input class="quantityinput form-control" value="0">' + '<button class="cancel btn btn-danger">Cancel</button>' + '</div>' + '<div class="subtotal col-xs-1">$0.00 </div>';
```

Comment:
Consider moving the code into multiple lines so it becomes easier to read also for you ;-)